### ISSUE TITLE
libudev-compat install vdev's headers

### ISSUE THREAD

**Issue by fbt** [jcnelson/vdev#30]
Should libudev-compat's makefile install vdev's headers?

Currently make -C libudev-compat install results in this in INCLUDEDIR:

```
-rw-r--r-- 1 fbt fbt 1.2K Jun 24 18:09 device-nodes.h
-rw-r--r-- 1 fbt fbt  18K Jun 24 18:09 hashmap.h
-rw-r--r-- 1 fbt fbt 2.5K Jun 24 18:09 hwdb-internal.h
-rw-r--r-- 1 fbt fbt 1.3K Jun 24 18:09 hwdb-util.h
-rw-r--r-- 1 fbt fbt 2.0K Jun 24 18:09 libudev-fs.h
-rw-r--r-- 1 fbt fbt  11K Jun 24 18:09 libudev.h
-rw-r--r-- 1 fbt fbt 9.9K Jun 24 18:09 libudev-private.h
-rw-r--r-- 1 fbt fbt 1.9K Jun 24 18:09 log.h
-rw-r--r-- 1 fbt fbt  21K Jun 24 18:09 macro.h
-rw-r--r-- 1 fbt fbt 1.6K Jun 24 18:09 mempool.h
-rw-r--r-- 1 fbt fbt 1.7K Jun 24 18:09 mkdir.h
-rw-r--r-- 1 fbt fbt 1.2K Jun 24 18:09 MurmurHash2.h
-rw-r--r-- 1 fbt fbt 1.8K Jun 24 18:09 sd-hwdb.h
-rw-r--r-- 1 fbt fbt 4.1K Jun 24 18:09 set.h
-rw-r--r-- 1 fbt fbt  409 Jun 24 18:09 siphash24.h
-rw-r--r-- 1 fbt fbt 3.7K Jun 24 18:09 sparse-endian.h
-rw-r--r-- 1 fbt fbt 1.5K Jun 24 18:09 strxcpyx.h
-rw-r--r-- 1 fbt fbt 2.1K Jun 24 18:09 utf8.h
-rw-r--r-- 1 fbt fbt  13K Jun 24 18:09 util.h
```

Which conflicts with already installed vdev's headers.

**Comment by jcnelson** [comment]
Hi fbt, thanks for point this out!  Can you confirm that it's fixed as of 3aaf0e332d9533e8001afc3a8654b2808063b91e?  Thanks!

**Comment by fbt** [comment]
There is a problem with that one:
On my system (Arch Linux), mksquashfs from squashfs-tools throws a syntax error when you do this:

    mksquashfs  -force-uid 0 -force-gid 0 "/home/fbt/git/pkg/vdev-git/src/vdev-git/build/lib/vdev/hwdb/hwdb.squashfs.dir" "/home/fbt/git/pkg/vdev-git/src/vdev-git/build/lib/vdev/hwdb/hwdb.squashfs"

It requires the options to go after the arguments, like this:

    mksquashfs "/home/fbt/git/pkg/vdev-git/src/vdev-git/build/lib/vdev/hwdb/hwdb.squashfs.dir" "/home/fbt/git/pkg/vdev-git/src/vdev-git/build/lib/vdev/hwdb/hwdb.squashfs" -force-uid 0 -force-gid 0
